test/integration/features/story_marketplace.feature

```diff
@@ -59,6 +59,19 @@ Feature:
         But executing "oc get pods $POD" fails
         And stderr matches "(.*)pods (.*) not found$"
 
+    @windows
+    Scenario: Failover
+        # simulate failure of 1 pod, check that it was replaced
+        When executing "$Env:POD = $(oc get pod -o jsonpath="{.items[0].metadata.name}")" succeeds
+        And executing "echo $Env:POD" succeeds
+        And executing "oc delete pod $Env:POD --now" succeeds
+        Then stdout should match "^pod(.*)deleted$"
+        # after a while 5 pods should be up & running again
+        And with up to "10" retries with wait period of "30s" command "oc get pods" output matches "(?s)(.*example-[a-z0-9]* *1/1 *Running.*){5}"
+        # but the deleted pod should not be up, it was replaced
+        But executing "oc get pods $Env:POD" fails
+        And stderr matches "(.*)pods (.*) not found$"
+
```

Comment:
@praveenkumar it would seem that if pods are deleted with `oc delete pods --all` (inside a namespace), then they don't come up again. So I'll need a bit more time to figure out how to kill them so they get revived... I'll update the PR when done.

Reply:
I can't delete all pods if they are to be restored. If quorum is lost, the only solution is to delete & create the etcd cluster again. There's some discussion about it here: https://github.com/coreos/etcd-operator/issues/1972. However, I will try to find a way to delete a pod without using it's full name (and not deleting all pods in the process). This should allow us to have one scenario for all platforms.

Reply:
@jsliacan Ack, in unix `oc get pod -o jsonpath="{.items[0].metadata.name}" | xargs oc delete pod` but that might not going to work with windows.

Reply:
> `oc get pod -o jsonpath="{.items[0].metadata.name}" | xargs oc delete pod`

@praveenkumar This didn't work on Windows (as you say), that's why I used the env vars in the current version. There's no way to kill a specific pod without referring to it by name. If you want to assign a label to a specific pod only, you also need to single it out first. Flags like `--keep-younger-than` is only available to `prune` command, which doesn't apply to pods. In short, I didn't find an elegant way to solve this in a platform-independent way. So I'm happy this is merged as it is. Thanks!